### ISSUE TITLE
Private documentation does not show inherited auth

### ISSUE THREAD

**Issue by abhijitkane** [a85/POSTMan-Chrome-Extension#4912]
My request has auth set to "inherit auth for parent". The parent collection has auth info saved, but viewing the documentation doesn't show any auth for the request or collection.

**Comment by steffenschebesta** [comment]
any updates on this issue?

**Comment by godfrzero** [comment]
Not quite yet. It'll be on the roadmap soon. And yes, this will also address #4244 @steffenschebesta

**Comment by steffenschebesta** [comment]
@godfrzero Thanks for your response. 
I know giving ETAs is sometimes hard, but can you give me a rough estimation whether this will be implemented a) within the next 3 months b) within 3 - 6 months c) within 6 - 12 months or d) in more than 12 months?
It's a crucial information for us to know since we're deciding whether or not to go with your solution (which is awesome in every other aspect so far). 
Thank you.

**Comment by steffenschebesta** [comment]
@godfrzero?

**Comment by godfrzero** [comment]
Ah, sorry. Like you said, giving an ETA is difficult, especially since there are multiple things already on the roadmap. I'd estimate 3-6 months for this particular feature right now, but that could change depending on what else pops up between now and then.

Either way, when we start implementing this features I'll be sure to update this thread.

**Comment by steffenschebesta** [comment]
@godfrzero 
I understand that you have multiple priorities atm. I just would like to point out that your automated API docs generation is currently completely useless for many very common authorization methods (sorry for the harsh words but it's the truth). And btw: that's a PRO feature that your customers are paying for...
Also, there's no workaround possible. I have tried out 3 methods to get the Authorization headers into the documentation. 
* Inheriting the auth from the parent: no headers show up in the docs; no auth headers in the example request on the right
* Setting the Authorization inside the request under "Authorization": a small lock icon shows up next to the request name; no auth headers in the example request on the right
* Setting the Authorization inside the request under "Headers": The header is shown in the middle column (at least); no auth headers in the example request on the right

So, in conclusion: Nothing works. The the third option (which is by far the worst) almost works. It's not pretty but if you could at least fix that, I could live with it for 3-6 months...

**Comment by steffenschebesta** [comment]
@godfrzero 
any updates on this issue? are you working on it?

**Comment by godfrzero** [comment]
The docs have already been updated to show the "lock" icon for inherited auth. As for displaying the actual information in the UI, we're still working on figuring out how to display it properly. It should be noted however, that when the collection is actually imported by anyone using the Run in Postman button, it will include the correct auth information.

CC @sankalp0o

**Comment by steffenschebesta** [comment]
@godfrzero any updates?

**Comment by SajidSalim** [comment]
Any updates on this?

**Comment by IvoPereira** [comment]
This is quite a drawback for who's using Postman for their API documentation methodologies.

Is there any new ETA on this?

**Comment by preethammavin** [comment]
Hi,
We have pushed a fix to support this, can you verify and let us know if it is resolved. Will keep the thread open for a while to see for the incoming feedback.

**Comment by miguejarias** [comment]
Still dealing with the same problem so it doesn't seem resolved unless i'm missing something.

**Comment by JP-from-Postman** [comment]
@preethammavin Appears to still be unresolved

**Comment by benfletcher** [comment]
Just ran into this. A clunky workaround is to manually add an `Authorization` header with `Bearer {{token}}`  as the content. But that doesn't allow any inheritance and changes require a ton of manual work.

I tested and you can leave the duplicate Authorization info without a conflict, so that when Postman fixes the issue we can hopefully just delete the manual header. (I've only tested Bearer auth with this approach.)

If you set Authorization to `none` and just have the manual header, the lock icon will go away in documentation, however.

Looking forward to the official fix.

**Comment by hellopablo** [comment]
I too am experiencing this.

Postman is fantastic for working with and maintaining an API, but this feels like a _crucial_ bit of information is missing from the generated docs. The lock is useful, but almost invisible - at the very least there're could be a section in a collection's meta (perhaps alongside the description) which is where authentication explanation could be added and the locks could link to that?

Perfect world, however, is that headers etc for authentication would be treated as first class citizens and shown alongside the manually defined headers for a request.

**Comment by heytimj** [comment]
A year and a half later and still happening on version 7.9.0. All requests in the collection use the same basic authorization so I have it configured at the collection level. All generated docs and code snippets are incorrect without including the inherited authorization headers. Any update?

**Comment by Ricardonacif** [comment]
I mean, this is a pretty basic functionality for teams...

**Comment by chrislentz** [comment]
Any updates on this? @godfrzero @sankalp0o @Raja-Simha 

It has been close to 2 years since originally reported. If I set authorization at the collection level and set all of my requests to "Inherit auth from parent", the auth headers don't show in the documentation at all.

**Comment by ranchanarajasekaran** [comment]
We have fixed this issue. Please feel free to reopen the ticket to report feedback on the same.

**Comment by eprothro** [comment]
I have a collection with header api key auth and requests in folders that both inherit the auth from parent. This works as expected in the postman client. 

I published my private documentation and do not see the auth header value included in the example requests.

The example in the [Postman API docs](https://documenter.getpostman.com/view/631643/JsLs/#8ca888b7-ef54-f3b4-312f-3f3e2e2cf04e) uses `No Auth` and explicitly includes the header value in each and every request. Is this what we're expected to do to get this working? 

If so, Id like to see this ticket reopened, since showing the inherited auth in the request documentation isn't all that helpful if the example requests don't show it as well.
